cds/modules/deposit/api.py

```diff
@@ -962,6 +963,24 @@ def discard(self, pid=None):
         )
         return video_discarded
 
+    @mark_as_action
+    def reserve_report_number(self):
+        """Reserve videos's report number until first publishing."""
+        #FIXME: to it better, cleaner
```

Comment:
do you need different permissions rather than simply `update_permission`?

Reply:
Anyone can update a deposit, if they have the correct ACL. But we want only the AVC team to be able to reserve report numbers, at least for now.

Reply:
Do we keep the FIXME??? you know that it will stay there forever!